api/data_explorer/controllers/facets_controller.py

```diff
@@ -51,9 +50,20 @@ def deserialize(filter_arr):
     for facet_filter in filter_arr:
```

Comment:
Sorry, some minor comments I should have made on previous PR
```
# filter_str looks like "Gender=male"
for filter_str in filter_arr:
    filter_str = urllib.unquote(filter_str).decode('utf8')
```

Reply:
Done